flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -283,12 +284,13 @@ static void _msp_send_analog(struct msp_bridge *m)
 		FlightBatterySettingsGet(&batSettings);
 	}
 
-	// TODO:  Verify these with a board that has working power stuff
 	if (batSettings.VoltagePin != FLIGHTBATTERYSETTINGS_VOLTAGEPIN_NONE)
 		data.status.vbat = (uint8_t)lroundf(batState.Voltage * 10);
 
-	if (batSettings.CurrentPin != FLIGHTBATTERYSETTINGS_CURRENTPIN_NONE)
-		data.status.current = lroundf(batState.Current * 10);
+	if (batSettings.CurrentPin != FLIGHTBATTERYSETTINGS_CURRENTPIN_NONE) {
+		data.status.current = lroundf(batState.Current * 100);
```

Comment:
The field names don't imply any particular unit and they're not (and I don't imagine ever will be) exported outside the scope of the module. I think within the scope of this module it's reasonable to assume that local variables will be in MultiWii units. To me current_100 is quite meaningless, does it mean this variable contains a value equivalent to 100 A in whatever scaling is being used, or current multiplied by 100, or current divided by 100, or...?